libfreerdp/codec/nsc.c

```diff
@@ -407,7 +407,7 @@ BOOL nsc_process_message(NSC_CONTEXT* context, UINT16 bpp,
 
 	if (!freerdp_image_copy(pDstData, DstFormat, nDstStride, nXDst, nYDst,
 				width, height, context->BitmapData,
-				PIXEL_FORMAT_BGRA32, 0, 0, 0, NULL))
+				PIXEL_FORMAT_BGRX32_VF, 0, 0, 0, NULL))
```

Comment:
@realjiangms Could you elaborate how you came to this color format (use case)? I've issues with h264 and windows 10.
See attached image, the menu icons are mirrored upside down
![screenshot from 2016-10-11 16 06 27](https://cloud.githubusercontent.com/assets/3472254/19273451/dd179194-8fcc-11e6-9da2-30d147da48fb.png)